test/operatortest.jl

```diff
@@ -10,11 +10,25 @@
 
 @assert coeffs(commutator(sigma_x, sigma_x)) == spzeros(2,2)
 
-######################################
-# Position & Momentum Operators Test #
-######################################
+####################################################
+# Position, Displacement & Momentum Operators Test #
+####################################################
 
 p = positionop(2)
 m = momentumop(2)
 @assert coeffs(commutator(sigma_x, p)) == spzeros(2,2)
 @assert coeffs(commutator(sigma_y, m)) == spzeros(2,2)
+
+csvd = coherentstatevec(20,1,method = "direct")
+csvo = displaceop(20,1)*statevec(1,FiniteBasis(20))
+@test_approx_eq_eps coeffs(csvd)[1] coeffs(csvo)[1] 1e-8
+@assert coherentstatevec(10,1.) == displaceop(10,1.)*statevec(1,FiniteBasis(10))
```

Comment:
This test is redundant since the right hand side is the definition of your function...